qpext/cmd/qpext/main.go

```diff
@@ -134,6 +219,7 @@ func scrape(url string, header http.Header, logger *zap.Logger) (io.ReadCloser,
 		return nil, cancel, "", fmt.Errorf("error scraping %s: %v", url, err)
 	}
 	if resp.StatusCode != http.StatusOK {
+		resp.Body.Close()
```

Comment:
hmm seems like we are not closing the `resp.Body` for the normal case after done reading the body?